### PR TITLE
Inittialize feedstock

### PR DESCRIPTION
**Pull request by marcoesters**
Create package for `configupdater` version 3.1.1.

[airflow 2.4.3](https://github.com/AnacondaRecipes/airflow-feedstock/pull/13) --> `configupdater`

Jira ticket: [PKG-574](https://anaconda.atlassian.net/browse/PKG-574)

Home/documentation: https://configupdater.readthedocs.io/
Development: https://github.com/pyscaffold/configupdater

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,17 +10,18 @@ source:
...
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools-scm
+    - wheel
   run:
-    - python >=3.6
+    - python
     - importlib-metadata
```
Comment (cbouss) [inline recipe/meta.yaml]: importlib is not required for recent versions
`- importlib-metadata # [py<38]`
https://github.com/pyscaffold/configupdater/blob/v3.1.1/setup.cfg#L53

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,17 +10,18 @@ source:
...
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skipt: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools-scm
```
Comment (cbouss) [inline recipe/meta.yaml]: The bounds could be added:
https://github.com/pyscaffold/configupdater/blob/v3.1.1/pyproject.toml#L3

**Review by cbouss — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (marcoesters) [reply recipe/meta.yaml]: Thank you! I fixed it

File: recipe/meta.yaml (hunk 2 above)
Comment (marcoesters) [reply recipe/meta.yaml]: I added the bounds to `setuptools` and `setuptools-scm`

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,18 +10,20 @@ source:
...
+  skipt: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
-    - setuptools-scm
+    - setuptools-scm >=5
+    - setuptools >=46.1.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: `toml` is missing https://github.com/pyscaffold/configupdater/blob/0539c207ac57ea4fa6a64618e41b987c5865ffd9/pyproject.toml#L3

Comment (marcoesters) [reply recipe/meta.yaml:22]: Added, thanks!

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**